README.md

```diff
@@ -166,12 +185,12 @@ To understand how the services interact, see `selfdrive/service_list.yaml`
 User Data / chffr Account / Crash Reporting
 ------
 
-By default openpilot creates an account and includes a client for chffr, our dashcam app. We use your data to train better models and improve openpilot for everyone.
+By default, openpilot creates an account and includes a client for chffr, our dashcam app. We use your data to train better models and improve openpilot for everyone.
 
 It's open source software, so you are free to disable it if you wish.
 
-It logs the road facing camera, CAN, GPS, IMU, magnetometer, thermal sensors, crashes, and operating system logs.
-The user facing camera is only logged if you explicitly opt-in in settings.
+It logs the road facing the camera, CAN, GPS, IMU, magnetometer, thermal sensors, crashes, and operating system logs.
```

Comment:
road-facing camera?
Other than that it looks good. Thanks

Reply:
revised. waiting for additional feedback. looking forward